packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/MarkersController.java

```diff
@@ -79,7 +99,10 @@ void hideMarkerInfoWindow(String markerId, MethodChannel.Result result) {
       markerController.hideInfoWindow();
       result.success(null);
     } else {
-      result.error("Invalid markerId", "hideInfoWindow called with invalid markerId", null);
+      result.error(
+          "Invalid markerId",
+          "hideInfoWindow called with invalid markerId or for hidden cluster marker",
```

Comment:
Consider including the markerId in this error message.

Reply:
Reverted the change, as just noticed that this changes the existing behaviour and is breaking change.
Eventhough it would be nice to give extra information for the developers that the error could be caused because marker is removed by the clusterManager, changing the existing API might not be an option here.